package.json

```diff
@@ -22,20 +22,21 @@
   "dependencies": {
     "@aeternity/aepp-components": "git+https://github.com/aeternity/aepp-components.git#develop",
     "@aeternity/aepp-components-3": "3.0.0-alpha.9",
-    "@aeternity/aepp-sdk": "6.0.0",
+    "@aeternity/aepp-sdk": "github:aeternity/aepp-sdk-js#aex-2",
```

Comment:
This branch is not up-to date with the Lima release and results in `Unable to connect to choosen node` error.

Reply:
Are the new changes working with the branch of the pull request to the sdk that supports aex-2 (https://github.com/aeternity/aepp-sdk-js/pull/512/)?

Reply:
The changes are working with fork of the sdk aex-2 branch. The only difference is changed node version because i was not able to connect to the testnet -> https://github.com/petbaik/aepp-sdk-js/blob/feature/aex-2/es/node.js#L158
` "@aeternity/aepp-sdk": "github:petbaik/aepp-sdk-js#feature/aex-2"`
The only difference between  sdk aex-2 branch and my fork is the changed node version